guides/tutorial/part-2/ember-data.md

```diff
@@ -127,21 +126,21 @@ We used the `@attr` decorator to declare the attributes of a rental property. Th
       "category": "Domaine",
       "bedrooms": 15,
       "image": "https://upload.wikimedia.org/wikipedia/commons/c/cb/Crane_estate_(5).jpg",
-      "description": "Ce manoir ancien et spatieux se trouve sur un domaine de plus de 100 acres de collines et de forêts de séquoias denses."
+      "description": "Ce manoir ancien et spacieux se trouve sur un domaine de plus de 100 acres de collines et de forêts de séquoias denses."
     }
   }
 }
 ```
 
-We can access these attributes for an instance of `RentalModel` using standard dot notation, such as `model.title` or `model.location.lat`. In addition to the attributes we declared here, there will always be an implicit _id_ attribute as well, which is used to uniquely identify the model object and can be accessed using `model.id`.
+On accède aux attributs d'une instance de `RentalModel` en utilisant une notation standard avec un point, comme `model.title` ou `model.location.lat`. En plus des attributs déclarés avec `@attr`, l'instance a toujours un attribut _id_ implicite, qui sert d'identifiant unique à l'objet modèles et auquel on peut accéder avec `model.id`.
 
-Model classes in Ember Data are no different than any other classes we've worked with so far, in that they allow for a convenient place for adding custom behavior. We took advantage of this feature to move our `type` logic (which is a major source of unnecessary duplication in our route handlers) into a getter on our model class. Once we have everything working here, we will go back to clean that up.
+Les classes `Model` de Ember Data ne sont pas différentes des autres classes avec lesquelles nous avons travaillé jusqu'ici, dans le sens où elles font un endroit idéal pour définir des comportements personnalisés. Par exemple, on a déplacé dans la classe `RentalModel` notre logique de `type` (qui était une source de duplication majeure et non nécessaire dans nos gestionnaires de route) à l'intérieur d'un _getter_ (accesseur). Une fois que tout fonctionnera ici, nous irons faire un peu de ménage.
```

Comment:
I would prefer `la classe est l'endroit idéal pour ...` over `la classe fait un endroit idéal pour ...`.

```suggestion
Les classes `Model` de Ember Data ne sont pas différentes des autres classes avec lesquelles nous avons travaillé jusqu'ici, dans le sens où elles sont l'endroit idéal pour définir des comportements personnalisés. Par exemple, on a déplacé dans la classe `RentalModel` notre logique de `type` (qui était une source de duplication majeure et non nécessaire dans nos gestionnaires de route) à l'intérieur d'un _getter_ (accesseur). Une fois que tout fonctionnera ici, nous irons faire un peu de ménage.
```

---

Not directly related, I see

> _getter_ (accesseur)
> EN term (FR term)

while I also see

> FR term (EN term)

which one should be used?

Here, should it be instead `accesseur (_getter_)` like in ```Le décorateur (_decorator_) `@attr` ``` from above?

Reply:
I think it's ok this way. "décorateur" is quite intuitive in French, but "getter" is more common than "accesseur". We have already used "_getter_ (accesseur)" in another file.